app/events/db/migrations/db_table_sql_scripts.sql

```diff
@@ -480,8 +480,7 @@ ALTER TABLE iiq_schema.basic_profile_info
 
 CREATE TABLE IF NOT EXISTS iiq_schema.dictionary_interests
 (
-    id          UUID      DEFAULT uuid_generate_v4() NOT NULL PRIMARY KEY,
-    name        VARCHAR(255)                                     NOT NULL,
+    name        VARCHAR(255) NOT NULL PRIMARY KEY,
```

Comment:
* every table should have an id column, which represents it's primary key
* name can not be treated as primary key, it can be unique column though